parse/parse.go

```diff
@@ -141,6 +143,41 @@ func setDefault(p *doc.Package, pi *PkgInfo, info types.Info) {
 	}
 }
 
+func setAliases(p *doc.Package, pi *PkgInfo, info types.Info) {
+	for _, v := range p.Vars {
+		for x, name := range v.Names {
+			if name != "Aliases" {
+				continue
+			}
+			spec := v.Decl.Specs[x].(*ast.ValueSpec)
```

Comment:
You need to make this `spec, ok :=`  and then check for ok, otherwise the next line will panic if you have a function named Aliases, for example.